### DISCUSSION TITLE
ngrok

### DISCUSSION THREAD

**Discussion by giscus** [yshmarov/yshmarov.github.io#13] [General]
# ngrok

Ngrok gives a public URL to your localhost. This is useful for testing from other devices, working with external APIs that require a real URL.

https://blog.corsego.com/ngrok

**Comment by secretpray** [comment]
config.hosts << /.*\.ngrok(-free)?\.app$/ # ngrok univaersal host

**Reply by secretpray** [reply]
# https://dashboard.ngrok.com/get-started/setup/macos
Static Domain
ngrok http --url=moving-snake-picked.ngrok-free.app 3000
